datetimeparser/evaluatorutils.py

```diff
@@ -67,23 +67,23 @@ def sanitize_input(current_time: datetime, parsed_list: list) -> list[Union[Rela
         """
 
         ps = parsed_list
-        for index, element in enumerate(parsed_list):
+        for i, element in enumerate(parsed_list):
```

Comment:
You assigned `parsed_list` to ps here but did not change it in the `enumerate` expression. Since those lists are being linked, I assumed you did this to get under a character limit or something else. In this case, remove it and keep the name `parsed_list` to improve readabillity